src/main/java/exception/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package exception;
```

Comment:
I like how the various exceptions are grouped differently (as opposed to just one generic `DukeException`).